mdtraj/geometry/order.py

```diff
@@ -189,6 +189,30 @@ def compute_inertia_tensor(traj):
     return A * eyes - B
 
 
+def compute_gyration_tensor(traj):
+    """Compute the gyration tensor of a trajectory.
+
+    For each frame,
+        S_{nm} = sum_{i_atoms} r^{i}_m r^{i}_n
```

Comment:
I think if you format this a bit differently, you could get it to show up in proper latex on the mdtraj.org website. e.g.: https://github.com/mdtraj/mdtraj/blob/master/mdtraj/geometry/internal.py#L373 or https://github.com/mdtraj/mdtraj/blob/e41a92299990fa6586a8f35739653a1801e33c31/mdtraj/geometry/hbond.py#L151

Reply:
what about also adding a link to the url of the nist.gov page you mentioned in the PR? that has a nice, concise, description.

Reply:
Done. I added a reference with the link in the doc string.